kirkeboeger_show3/index.php

```diff
@@ -199,23 +199,11 @@ function minusTen() {
 
    <!-- My own buttons after -->
     <!-- +1 page -->
-<div class="flex-item"><input type="button" button title="1 side frem"style="width: 30px; margin-right:3px; text-align:center; color: #f6f6f6; background-color:#666666; margin-left:20px; border:none";id="oneplus"; onclick="plusOne()" value="1>"></div>
-
- <script>
-function plusOne() {
-    document.getElementById("oneplus").innerHTML = viewer.goToPage(Math.min(viewer.tileSources.length - 1, viewer.currentPage() + 1)); // 1 pages after;
-}
-</script>
+<div class="flex-item"><input type="button" button title="1 side frem"style="width: 30px; margin-right:3px; text-align:center; color: #f6f6f6; background-color:#666666; margin-left:20px; border:none";id="oneplus"; onclick="changePage(1)" value="1>"></div>
```

Comment:
You can see I've now made a single function that you can call from all of your buttons with different values. I've done the forward 1 and forward 5; you can do the rest.